invest/helpers.py

```
@@ -0,0 +1,32 @@
+from urllib.parse import urljoin
+from django.conf import settings
+import requests
+
+
+class IPStackAPIClient:
+    session = requests.Session()
+    api_key = settings.IPSTACK_API_KEY
+    base_url = 'http://api.ipstack.com'
+    auth = '?access_key=' + api_key
+
+    @classmethod
+    def get(cls, ip):
+        url = urljoin(cls.base_url, ip) + cls.auth
+        response = cls.session.get(url)
+        data = response.json()
```

if the response is 500 for ecample then the response will not be valid json, and this will become an uncaught excepotion. is that ok, or do we want to return `None` if that happens?

---

We should probably silently fail for 500 as well. The code below handles the errors https://ipstack.com/documentation